server/tserver/src/main/java/org/apache/accumulo/tserver/log/LogSorter.java

```diff
@@ -61,8 +61,8 @@ public class LogSorter {
   VolumeManager fs;
   AccumuloConfiguration conf;
 
-  private final Map<String,LogProcessor> currentWork =
-      Collections.synchronizedMap(new HashMap<String,LogProcessor>());
+  // access must synchronize on currentWork to guard against concurrent updates
+  private final Map<String,LogProcessor> currentWork = new HashMap<>();
```

Comment:
Unless we're fixing a bug, I'd rather not risk introducing a new bug in 1.10 by changing concurrency assumptions in a patch release. Is this actually a bug, or just unneeded? I don't think it's a performance issue in here.

Reply:
I set the base to 3.0 - we could included it in 2.1.1, but it seems that would have the same objection.